tests.spec

```diff
@@ -0,0 +1,21 @@
+Name:       tests
+Version:    1
+Release:    1
+Summary:    Dummy just to trigger tests
+License:    MIT
+BuildArch:  noarch
```

Comment:
Looks like we can't work without a source to be specified. I've created an issue for that: https://github.com/packit/packit/issues/1251

The source will be replaced by the archive of this repository (by default) so the content of that variable should not be something meaningful.

Reply:
There is a workaround for that -- see the other comment.

Reply:
Thanks for helping me out but the problem is still there.
```
Message: Preparation of the repository for creation of an SRPM failed: min() arg is an empty sequence
Exception: PackitSRPMException('Preparation of the repository for creation of an SRPM failed: min() arg is an empty sequence')
```
https://dashboard.packit.dev/results/srpm-builds/24382

Reply:
OK, there is another place we work with sources. To overcome this, you can add a `sources` directory. Its existence will turn off downloading of the sources.

With that change, I've been able to build `SRPM` via packit locally.

Reply:
I can see tests to be running \o/ glad you guys solved this out

Reply:
Yep, thank you for you assistance.

There's still some issue as it reports it is still running even if the test is already finished.

Reply:
that will be fixed on monday